egs2/css10/tts1/conf/vad.conf

```diff
@@ -0,0 +1,4 @@
+--vad-energy-threshold=5.5
+--vad-energy-mean-scale=0.5
+--vad-proportion-threshold=0.12
+--vad-frames-context=2
```

Comment:
This file may be removed.

Reply:
This is need if we want to extract x-vector.